src/models/backbone.py

```diff
@@ -0,0 +1,36 @@
+from transformers import BertConfig, BertForSequenceClassification
+from torch import nn
+from loguru import logger
+
+
+def get_model(
+    vtr: bool,
+    vocab_size: int = 30_000,
+    max_position_embeddings: int = 512,
+    hidden_size: int = 768,
+    num_attention_heads: int = 12,
+    num_hidden_layers: int = 1,
+    type_vocab_size: int = 1,
+    dropout: float = 0.0,
+    num_classes: int = 2,
+):
+    if vtr:
+        logger.info(f"Initializing VTR classifier | hidden size: {hidden_size}, # layers: {num_hidden_layers}")
+        return nn.Linear(hidden_size, num_classes)
```

Comment:
This is strange, regardless embedder, backbone should be the same: VTR + BertForSequenceClassification or TTR + BertForSequenceClassification